stories110M/wasm/export.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+temp_pt="$(mktemp).pt"
+temp_params="$(mktemp).json"
+
+wget -O "$temp_pt" "https://huggingface.co/karpathy/tinyllamas/resolve/main/stories110M.pt"
+wget "https://raw.githubusercontent.com/karpathy/llama2.c/master/tokenizer.model"
+echo '{"dim": 768, "multiple_of": 32, "n_heads": 12, "n_layers": 12, "norm_eps": 1e-05, "vocab_size": 32000}' > "$temp_params"
```

Comment:
are the params not available on HF? Are they not checked in anywhere accessible?

If they arent available anywhere else then this seems fine. Its not like they are any more fragile then the hf path itself. It just seems like it would be better to pull them from a real file if they are checked in anywhere